spot/plugins/SkyCam.py

```diff
@@ -36,49 +103,112 @@ def __init__(self, fv, fitsimage):
         # superclass defines some variables for us, like logger
         super().__init__(fv, fitsimage)
 
+        self.viewer = self.fitsimage
+        self.dc = fv.get_draw_classes()
+        self.std = np.array([0.2126, 0.7152, 0.0722])
+        self.old_data = None
+        self.cur_data = None
+        self.img_src_name = 'Subaru Telescope (Visible)'
+
+        loaddir = ginga_home
+        path = os.path.join(loaddir, "cameras.yml")
+        if os.path.exists(path):
+            with open(path, 'r') as cam_f:
+                self.configs = yaml.safe_load(cam_f)
+        else:
+            # Sets Subaru Telescope (Visible) as default
+            # if cameras.yml does not load
+            self.config = {
+                'url': "https://allsky.subaru.nao.ac.jp/allsky/api/v0/images/download_most_recent.png",
+                'file': "allsky.png",
+                'lfile': "allsky-%ld.png",
+                'title': "Subaru Telescope (Visible)",
```

Comment:
What is the purpose of ``file`` and ``lfile`` in config?  They don't seem to be used in the download part?